styleguide/source/assets/scss/01-atoms/_listicle.scss

```diff
@@ -0,0 +1,52 @@
+ol.listicle {
+  counter-reset: ama-listicle;
+  list-style: none;
+  @include gutter($padding-top-half...);
+  @include gutter($padding-bottom-half...);
+
+  > li {
+
+    counter-increment: ama-listicle;
+    position: relative;
+    @include gutter($margin-bottom-half...);
+    @include gutter($margin-top-half...);
+
+    &::before {
+      content: counter(ama-listicle);
+      position: absolute;
+      background: $homepagePurple;
+      color: $white;
+      width: 30px;
+      height: 30px;
+      top: 5px;
+      text-align: center;
+      font-size: 1.5rem;
+      display: flex;
+      align-items: center;
+      justify-content: center;
+
+      @include type($myriad-pro, $font-weight-bold);
+    }
+
+    > ol {
+      list-style: none;
+      margin-left: 55px;
+      padding-left: 0;
+
+      > li {
+        @include gutter($margin-bottom-full...);
+        @include gutter($margin-top-full...);
+      }
+    }
+
+    h2 {
```

Comment:
use class instead of h2

the H2 should be:
`font-size: 22px;
  font-weight: 600;`